### ISSUE TITLE
How to evaluate the model?

### ISSUE THREAD

**Issue by rezacsedu** [amanrana20/Diabetic_Retinopathy_Detection#2]
Hi,

Thanks for the code! How to evaluate the trained model? I don't see any code for that!

**Comment by motatoes** [comment]
Yes if you could share the evaluation or testing code to produce predictions that would be helpful @amanrana20 . Also it would be helpful to add code to save/pickle the model after the training has done so that the test code can load the model and perform predictions.

I can contribute bits of code if you can give me an idea of how to do it -- I am new to TF
